server/controllers/applicationsController.ts

```diff
@@ -174,10 +174,48 @@ const updateApplication = async (
   }
 };
 
+const getAggregatedUserStats = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    const userId = req.query.user_id;
+
+    const totalApplicationsQuery =
+      "SELECT COUNT(*) FROM applications WHERE user_id = $1";
+    const totalApplicationsResult = await pool.query(totalApplicationsQuery, [
+      userId,
+    ]);
```

Comment:
Do we need this query, if we are going to get all of the applications in the next query? Can't we get length and not have to do 2 database calls?

Reply:
Great call - New code pushed up!